ipq40xx: fix USB on Aruba AP-303H

ryanrsrs opened the pull request:
Disable USB 2.0 controller, enable USB 3.0 controller in device tree.

The USB 2.0 port on the AP-303H is actually connected to the USB 3.0 controller's HS phy. The SS phy must also be enabled, even though the SuperSpeed lanes aren't brought out to the connector.

robimarko commented:
There is no need to enable the SS PHY if its not actually wired up, look at AP120C which also has only HS PHY connected to USB3 controller, you just need to override the default node.

Also, can you please move to using labels for USB instead of having to deal with full path under soc?

robimarko commented:
@ryanrsrs 

For the USB3 DWC PHY tweaks instead of:
`dwc3@8a00000` under the USB3 node just use its phandle of `&usb3_dwc` like `&usb3` is used for kernel 6.6, for 6.1 you have to do what you are already doing as there is no label for it.

So:
```
&usb3_dwc {
		phys = <&usb3_hs_phy>;
		phy-names = "usb2-phy";
	};
```

ryanrsrs commented:
Use usb3_dwc label for 6.6.

ryanrsrs committed:
ipq40xx: fix USB on Aruba AP-303H

Enable USB 3.0 controller, disable USB 2.0 controller.

The USB 2.0 port on the AP-303H is actually connected to the USB 3.0
controller's HS phy. Enable the HS phy only, since the SS lanes are not
brought out to the connector.

Signed-off-by: Ryan Salsbury <ryanrs@gmail.com>

robimarko commented:
Thanks! Rebased on top of main and merged!
